EventFilter/CSCRawToDigi/interface/CSCDigiToRawAccept.h

```diff
@@ -0,0 +1,85 @@
+#ifndef EventFilter_CSCRawToDigi_CSCDigiToRawAccept_h
+#define EventFilter_CSCRawToDigi_CSCDigiToRawAccept_h
+
+/** \class CSCDigiToRawAccept
+ *
+ * Static class with conditions to accept CSC digis in the Digi-to-Raw step
+ *
+ *  \author Sven Dildick - Rice
+ */
+
+#include "DataFormats/CSCDigi/interface/CSCALCTDigiCollection.h"
+#include "DataFormats/CSCDigi/interface/CSCCLCTDigiCollection.h"
+#include "DataFormats/CSCDigi/interface/CSCCLCTPreTriggerCollection.h"
+#include "DataFormats/CSCDigi/interface/CSCCLCTPreTriggerDigiCollection.h"
+
+class CSCDigiToRawAccept {
```

Comment:
Is this class intended to store any state? What motivates the choice of a static class vs. the previous namespace?

Reply:
I can replace it with a namespace if that's more desirable.